a10_octavia/api/drivers/driver.py

```diff
@@ -241,6 +241,18 @@ def get_supported_flavor_metadata(self):
     def validate_flavor(self, flavor_dict):
         try:
             validate(flavor_dict, flavor_schema.SUPPORTED_FLAVOR_SCHEMA)
+
+            # validate nat-pool and nat-pool-list keys
+            if 'nat-pool' in flavor_dict:
+                nat = flavor_dict['nat-pool']
+                if (('pool-name' not in nat or 'start-address' not in nat or
+                     'end-address' not in nat or 'netmask' not in nat)):
+                    raise Exception('nat-pool manditory key missing')
```

Comment:
Can be : `raise Exception('nat-pool required in flavor schema')`

Reply:
ok, will change to pool-name/start-address/end-address/netmask is required in nat-pool-list flavor